README.md

```diff
@@ -612,10 +613,19 @@ backend.useMiddleware('readSnapshots', function (request, callback) {
   var connectionInfo = request.agent.custom;
   var snapshots = request.snapshots;
 
-  // Use the information provided at connection to determine if a user can access snapshots.
+  // Use the information provided at connection to determine if a user can access the snapshots.
   // This should also be checked when fetching and submitting ops.
-  if (!userCanAccessSnapshots(connectionInfo, snapshots)) {
-    return callback(new Error('Authentication error'));
+  if (!userCanAccessCollection(connectionInfo, request.collection)) {
+    return callback(new Error('Not allowed to access collection ' + request.collection));
+  }
+  // Check each snapshot individually.
+  // In ES6, this could be `for (const snapshot of snapshots)`
+  for (var i = 0; i < snapshots.length; i++) {
```

Comment:
Since this example code is server only, we can write in ES6 syntax. I wouldn't worry about the fact that sharedb is still in ES5 style